calling out that Huawei app is out of date (#8228)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/get-teams-android-in-china.md

|---------|---------|
| Lenovo: https://aka.ms/TeamsLenovo      | ![QR code for Teams on Android from Lenovo store.](media/get-teams-android-in-china-lenovo.png)        |
| Xiaomi: https://aka.ms/TeamsXiaomi     |![QR code for Teams on Android from Xiaomi store.](media/get-teams-android-in-china-xiaomi.png)         |

> |Huawei: https://aka.ms/TeamsHuawei  NOTE: The Teams app in the Huawei app store is several months out of date and missing many features and security updates. Microsoft recommends that Huawei customers use a different phone or update the app from a different app store.   | ![QR code for Teams on Android from Huawei store.](media/get-teams-android-in-china-huawei.png)        |

Reviewer: ```suggestion
|Huawei: https://aka.ms/TeamsHuawei  **NOTE**: The Teams app in the Huawei app store is several months out of date and missing many features and security updates. We recommend that Huawei customers update the app from a different app store.   | ![QR code for Teams on Android from Huawei store.](media/get-teams-android-in-china-huawei.png)        |
```
